app/code/community/Bolt/Boltpay/Model/Order.php

```diff
@@ -329,7 +279,69 @@ public function createOrder($reference, $sessionQuoteId = null, $isPreAuthCreati
         return $order;
     }
 
+    /**
+     * @param $immutableQuote
+     * @param $transaction
+     * @param $isPreAuthCreation
+     * @return bool
```

Comment:
If we are going to return a value, please explain what the value being returned is.